components/script/dom/formdata.rs

```diff
@@ -163,4 +164,35 @@ impl FormData {
 
         ret
     }
+
+    pub fn atoms(&self) -> Vec<Atom> {
+        let mut ret = vec![];
+        for (key, value) in self.data.borrow().iter() {
```

Comment:
Sure, I love the amazing `Iterators`. I thought the code in the original PR is what Servo codebase prefer since I saw the existing `datums` function above. It looks like I'm wrong.

```
pub fn datums(&self) -> Vec<FormDatum> {       
    let mut ret = vec![];                      
    for values in self.data.borrow().values() {
        ret.append(&mut values.clone());       
    }                                          
                                               
    ret                                        
}                                              
```

 I'll modify the `atoms` function.